addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -266,7 +266,7 @@ function factory(dependencies) {
                     channel.markAsFetched();
                 }
                 // (re)open chat on receiving new message
```

Comment:
adapt comment, it's no longer "(re)open" but only "open ... if it was not already opened or folded"

Reply:
@seb-odoo Done.